adafruit_azureiot/iothub_device.py

```diff
@@ -166,30 +174,32 @@ def __init__(
         self._logger.debug("Device Id: " + self._device_id)
         self._logger.debug("Shared Access Key: " + self._shared_access_key)
 
-        self._on_connection_status_changed = None
-        self._on_direct_method_invoked = None
-        self._on_cloud_to_device_message_received = None
-        self._on_device_twin_desired_updated = None
-        self._on_device_twin_reported_updated = None
+        self._on_connection_status_changed = lambda *x: None
+        self._on_direct_method_invoked = lambda *x: None
+        self._on_cloud_to_device_message_received = lambda *x: None
+        self._on_device_twin_desired_updated = lambda *x: None
+        self._on_device_twin_reported_updated = lambda *x: None
```

Comment:
I'm not familiar with the syntax on these ones. I don't think they were flagged when mypy was run originally. Can you tell me a bit about it, or know a resource I could look at?

Reply:
They weren't flagged in the initial issue. I think I had a mypy run (before I asked for the specific command ran) that flagged `None` as incompatible with the return type `Callable`. Since all these functions were callbacks, I modified them to be a lambda that takes as many arguments as passed and does nothing to emulate a callback function.

StackOverflow reference: https://stackoverflow.com/questions/22738412/a-suitable-do-nothing-lambda-expression-in-python

I can revert these back to `None` if you'd like.